connectivity/cellular/include/cellular/framework/API/CellularDevice.h

```diff
@@ -360,6 +360,12 @@ class CellularDevice {
      */
     void set_plmn(const char *plmn);
 
+    /** Get the plmn which is used is manual registering. If plmn is null then automatic registering is used.
+     *
+     *  @return plmn is it's set or null
+     */
+    const char *get_plmn() const;
```

Comment:
This cellular addition should be done in separate pull request (adding new functionality).

Reply:
Thanks for your comments. 
The code for supporting s5js100 target depends on the code for new cellular functionality.
Shall I send the pull request about new cellular features first? and then send the PR about s5js100 target?
Otherwise, the PR about s5js100 can't pass the CI.

Reply:
yes please. Mark it as dependency and we can review accordingly.